Source/WebCore/platform/graphics/gstreamer/eme/WebKitCommonEncryptionDecryptorGStreamer.cpp

```diff
@@ -363,4 +365,19 @@ static void webKitMediaCommonEncryptionDecryptDefaultReleaseCipher(WebKitMediaCo
 {
 }
 
+static gchar* getContentType(GstElement* element)
+{
+    gchar* contentType = nullptr;
+
+    GstPad* sinkPad = gst_element_get_static_pad(element, "sink");
+    GstCaps* elementCaps = gst_pad_get_current_caps(sinkPad);
```

Comment:
You need to do the same smart pointer changes as for the other twin function.

Reply:
Resolved